tka/tailchonk.go

```diff
@@ -596,7 +596,7 @@ func markActiveChain(storage Chonk, verdict map[AUMHash]retainState, minChain in
 				// We've reached the end of the chain we have stored.
 				return h, nil
 			}
-			return AUMHash{}, fmt.Errorf("reading active chain (retainStateActive) (%d): %w", i, err)
+			return AUMHash{}, fmt.Errorf("reading active chain (retainStateActive) (%d, %s): %w", i, parent, err)
```

Comment:
Any reason to use `%s` rather than `%v` here? I'd suggest being consistent, and `%v` seems to work well for `AUMHash`.